www/choosing-install-method.rst

```diff
@@ -0,0 +1,149 @@
+.. _choosing-install-method:
+
+Choosing an install method
+==========================
+
+There are many ways to install Python itself and the SciPy Stack.  This
+section attempt to give a concise summary of the pros and cons of the
+most common methods, and a few guidelines on how to pick the best method
+for you.  Note that the actual install instructions for each method
+can be found in :ref:`Installing the SciPy Stack <install>`.
+
+The main methods are:
+
+- A scientific Python distribution
+- Using an OS specific package manager
+- Using pip_
+- Using conda_
+
+
+Pros and cons per method
+------------------------
+
+A scientific Python distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Pros:
+
+- A single download for everything, including Python itself.
+- All scientific Python packages will have been tested to work together.
+- Often a GUI for updates and installs is provided (in addition to the normal
+  command-line tools).
+- A very large collection of scientific Python packages, as well as common
+  web frameworks and other popular Python packages.
+- Also includes important non-Python packages (like HDF5, Qt, gcc, R, etc.) that
+  are often hard to install by other means.
+- Binaries for everything, usually no compiler needed.
+
+Cons:
+
+- There may not be an installer if you're on a less commonly used OS
+  (32-bit Linux for example).
+- Doesn't integrate with an existing Python installation.
+
+Using an OS-specific package manager
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Pros:
+
+- Package manager itself already installed (on Linux) or so useful for
+  non-Python things that you'll want to install it anyway (on OS X).
+- Package managers are reliable, mature technology.
+- Easy to get started.
+
+Cons:
+
+- On Linux, installs into the same Python environment as used by the OS itself,
+  which can be fragile if you want to install packages also with pip_.
+- On OS X, packages provided by Homebrew and Macports are often not as well
```

Comment:
My reasoning for recommending pip over homebrew was that the pip wheels cover more packages, so it's easier to stick to one method rather than mixing homebrew and pip installs.   I didn't mean to suggest the homebrew installs were poorly tested or optimized, I don't have any reason to think that.